src/components/services/Ping.vue

```diff
@@ -29,7 +29,8 @@ export default {
   },
   methods: {
     fetchStatus: async function () {
-      this.fetch("/", { method: "HEAD", cache: "no-cache" }, false)
+      const method = typeof this.item.method === 'string' && this.item.method.toLowerCase() === "get" ? "GET" : "HEAD"
```

Comment:
I would prefer to check if the method is valid and stop if not. No unexpected behavior this way. Something like this: 
```js
const method = this.item.method.toUpperCase();
if (!['GET', 'HEAD', 'OPTION'].includes(method)) {
  console.error(`invalid ${method} ping method`);
  return;
}
```
Would be great.